src/components/CardModal.tsx

```diff
@@ -17,33 +16,41 @@ function CardModal({ image, isLikedImage, isOpen, onLike, onClose }: CardModalDa
 
   return (
     <>
-      <div onClick={onClose} className="blackout_container" />
+      <div onClick={onClose} className="fixed inset-0 z-20 bg-modal_blackout" />
 
-      <div className="modal_container">
-        <div className="cm_content">
-          <span onClick={onClose} className="close_modal_button">
+      <div className="fixed z-30 h-auto bottom-1/2 right-1/2 transform translate-x-1/2 translate-y-1/2 bg-navy_blue border border-cream max-w-[56rem] rounded-md w-[83.333333%] overflow-hidden">
+        <div className="h-full flex flex-col">
+          <span onClick={onClose} className="absolute z-10 right-3 font-[500] text-lg cursor-pointer">
             X
           </span>
 
-          <div className="cm_image_container">
+          <div className="relative h-36 flex-grow hover:overflow-hidden">
             <Link to={highDefSrc || ''} target="_blank" rel="noreferrer">
-              <img src={src} className="cm_img" />
+              <img
+                src={src}
+                className="h-full opacity-70 rounded-bl-md rounded-br-md transition-all ease-in-out duration-700 border-none hover:transform hover:scale-110"
+              />
             </Link>
-            <span className="cm_title">{title}</span>
-            <span className="cm_author">{copyright}</span>
-            <span className="like_button" onClick={onLike}>
+            <span className="absolute top-[6.5rem] bottom-0 left-0 right-0 w-full font-normal text-lg text-center flex justify-center items-center md:text-xl">
+              {title}
+            </span>
+            <span className="absolute top-1 left-1 text-xs opacity-70">{copyright}</span>
+            <span
+              className="absolute m-2 right-0 bottom-1 cursor-pointer font-[500] h-[1.3rem] w-[1.3rem] border-none hover:opacity-75"
+              onClick={onLike}
+            >
               <img
-                className="noborder"
+                className="border-none"
                 src={`${isLikedImage ? 'media/liked-heart-icon.png' : 'media/unliked-heart-icon.png'}`}
               />
             </span>
           </div>
 
-          <div className="cm_body">
-            <span className="cm_date">{new Date(date!).toDateString().slice(4)},</span>
+          <div className="flex flex-col flex-grow justify-start pt-4 border-t indent-5">
+            <span className="text-lg text-left">{new Date(date!).toDateString().slice(4)},</span>
 
-            <span className="cm_salutations">A message from the astronomer,</span>
-            <p className="cm_explanation">{explanation}</p>
+            <span className="text-left pb-1 text-lg">A message from the astronomer,</span>
+            <p className="m-5 h-40 flex-grow overflow-auto text-center indent-5 leading-[1.5rem]">{explanation}</p>
```

Comment:
`text-justify` for this paragraph looks better imho. Also maybe right padding for a bit of spacing from the scrollbar
```suggestion
            <p className="m-5 h-40 pr-2 flex-grow overflow-auto text-justify indent-5 leading-[1.5rem]">{explanation}</p>
```

Reply:
Its really difficult on the eyes as it right now. I agree this new way looks much better & easier to read. Appreciate it!